### DISCUSSION TITLE
Full stack development in Go

### DISCUSSION THREAD

**Discussion by mpostument** [TheOdinProject/curriculum#26073] [Ideas]
Go is very popular right now. Do you know if there is a plan to add a tutorial for it?

**Comment by Arkanheal** [comment]
As much as I love Golang, it'd been a lot of extra work to maintain 🤔.

**Comment by CouchofTomato** [comment]
Thanks for the discussion.

As has been mentioned, the biggest issue is maintaining it. We just don't have the resources to build an entire new track right now.
